Assets/Prefabs/Core/Player.prefab

```diff
@@ -222,7 +220,7 @@ MonoBehaviour:
     m_Bits: 256
   collisionDistance: 0.1
   wallCloseDistance: 2
-  maxGroundSpeed: 20
+  maxGroundSpeed: 22
```

Comment:
To future reviewers, the only changes are `maxGroundSpeed` 20 -> 22 and hack velocity ratio set to default